data/vendor-customer-support.ini

```diff
@@ -0,0 +1,8 @@
+[Customer Support]
```

Comment:
Yes. That would let me remove one function, but there would be some disadvantages:

* Translators become responsible for keeping the contact info correct for each locale.
* Unless you have completely separate es_GT and es_MX translations (I suspect not), then you'd need to display all the numbers for both countries. Probably fine with just three numbers, but if you add more countries in the future....
* This way, the numbers can be kept all in one place, and easily updated or reviewed for correctness, without digging into different po files.

This file could also move to a different package and be used by different components that might want to display customer support contact info.